frontend/src/components/Comments/Comments.jsx

```diff
@@ -75,9 +76,19 @@ export default function Comments({ ticker }) {
                 {comments?.map((comment) => {
                     return (
                         <div className="comment-details" key={comment.content}>
-                            <p className="comment-name">{capitalize(comment.User.username)}</p>
-                            <p className="comment-date">{formatDateTime(comment.createdAt)}</p>
-                            <p className="comment-content">{comment.content}</p>
+                            <div className="comment-picture">
+                                <img
+                                    src={comment.User.picture}
+                                    alt={`This is a profile picture associated with ${comment.User.username}`}
+                                />
+                            </div>
+                            <div className="comment-content">
+                                <Link to={`/profile/${comment.User.username}`} className="user-link">
```

Comment:
It's better not to assume that the comment object has a User object, that has a username prop. 
The safer way is `comment?.User?.username ?? ''`

Reply:
Nice Link 👍

Reply:
It always has a User object though.